product_subscription/models/subscription_object.py

```diff
@@ -65,6 +70,26 @@ def _compute_renewed(self):
             )
             subscription.renewed = True if subscriptions else False
 
+    @api.multi
+    @api.depends("state", "is_trial", "subscribed_on", "subscriber")
+    def _compute_is_first_subscription(self):
+        for subscription in self:
+            if subscription.is_trial or subscription.state in (
+                "draft",
+                "cancel",
+            ):
+                subscription.is_first_subscription = False
+            else:
+                other_subs = self.search(
+                    [
+                        ("subscriber", "=", subscription.subscriber.id),
+                        ("start_date", "<", subscription.start_date),
+                        ("state", "not in", ("draft", "cancel")),
+                        ("is_trial", "!=", True),
+                    ]
+                )
+                subscription.is_first_subscription = not bool(other_subs)
```

Comment:
be kind with the possible maintainers and add a bit of explanation :)

Reply:
done